infra/terraform/templates/alb.tf

```diff
@@ -9,20 +9,21 @@ resource "aws_alb" "wellcomecollection_alb" {
   security_groups = [
     "${aws_security_group.https.id}",
     "${aws_security_group.http.id}",
-    "${aws_security_group.node_app_port.id}"
+    "${aws_security_group.node_app_port.id}",
+    "${aws_security_group.docker.id}"
   ]
 }
 
 resource "aws_alb_target_group" "wellcomecollection" {
   name     = "wellcomecollection"
-  port     = 3000
+  port     = 80
   protocol = "HTTP"
   vpc_id   = "${aws_vpc.wellcomecollection.id}"
 
   health_check {
-    healthy_threshold   = 2
-    unhealthy_threshold = 10
-    timeout             = 20
+    healthy_threshold   = 5
+    unhealthy_threshold = 2
+    timeout             = 5
```

Comment:
`interval` = time between health checks.
`un/healthy_threshhold` = the amount of consecutive passed / failed health checks until reported un/health.
`timeout` = the amount of time to wait for a response before it's considered unhealthy.

So this does a health check every 30 seconds, waiting 5 seconds for a response. We require 5 responses to be healthy, and 5 timeouts to be unhealthy.

These are the defaults.